phpunit.xml

```diff
@@ -9,7 +9,6 @@
          processIsolation="false"
          stopOnError="false"
          stopOnFailure="false"
-         syntaxCheck="true"
```

Comment:
Could you explain this?

Reply:
In PHPUnit 8 it produces a warning as an unrecognised parameter – I think in previous versions it was just ignored (but still did nothing). 

https://stackoverflow.com/questions/44328114/phpunit-what-does-syntaxcheck-configuration-parameter-stands-for-exactly/44331140#44331140

Reply:
Ok thanks!